.github/workflows/pipeline-queuetime.yml

```diff
@@ -0,0 +1,59 @@
+name: Build Helm Charts
+on:
+  workflow_call:
+    inputs:
+      workflow_name:
+        type: string
+        required: true
+        description: "Name of the calling workflow for easy differentiation between metric datapoints"
+    secrets:
+      monitoring_api_token:
+        required: true
+      monitoring_tenant_url:
+        required: true
+defaults:
+  run:
+    shell: bash
+jobs:
+  calculate-queuetime:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Calculate queue time
+        id: calculate_queue_time
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const repo = "keptn";
+
+            const workflowRunReponse = await github.rest.actions.getWorkflowRun({
+              owner: repo,
+              repo: repo,
+              run_id: context.runId
+            });
+
+            const jobResponse = await github.rest.actions.listJobsForWorkflowRun({
+              owner: repo,
+              repo: repo,
+              run_id: context.runId
+            });
+            const workflowRun = workflowRunReponse.data;
+            const jobs = jobResponse.data.jobs;
+
+            const runStartedTime = new Date(workflowRun.created_at).getTime();
+            const jobStartedTime = new Date(jobs[0].started_at).getTime();
+
+            const queueTime = (jobStartedTime - runStartedTime) / 1000;
+            return queueTime;
+
+
+      - name: Ingest Pipeline Metrics
+        uses: dynatrace-oss/dynatrace-github-action@v8
+        with:
+          url: ${{ secrets.monitoring_tenant_url }}
+          token: ${{ secrets.monitoring_api_token }}
+          metrics: |
+            - metric: "github.workflow_queue_time"
+              value: "${{ steps.calculate_queue_time.outputs.result }}"
+              dimensions:
```

Comment:
I recommend adding another dimension here: reponame (e.g., `keptn/keptn`). This will make it easier to filter in the monitoring solution.

Reply:
sure, i will add that